requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests>=0.34
```

Comment:
Request is needed only for `build_custom_app.py`.
It is already in the `requirements_build_custom_app.txt` (https://github.com/kiwix/kiwix-build/blob/master/requirements_build_custom_app.txt) file.